product_assortment/views/product_assortment.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+    <record id="product_assortment_view_search" model="ir.ui.view">
+        <field name="name">product.assortment.search</field>
+        <field name="model">ir.filters</field>
+            <field name="arch" type="xml">
+                <search string="Assortments">
+                    <field name="name" string="Assortment Name"/>
+                    <field name="model_id"/>
+                </search>
+            </field>
+    </record>
+
+    <record id="product_assortment_view_tree" model="ir.ui.view">
+        <field name="model">ir.filters</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+            </tree>
+        </field>
+    </record>
+
+    <record id="product_assortment_view_form" model="ir.ui.view">
+        <field name="model">ir.filters</field>
+        <field name="arch" type="xml">
+            <form string="Filters">
+                <sheet>
+                    <div class="oe_button_box" name="buttons">
+                        <button class="oe_stat_button"
+                                name="show_products" type="object">
+                            <div class="o_stat_info">
+                                <field name="record_count" class="o_stat_value"/>
+                                <span class="o_stat_text">Products</span>
+                            </div>
+                        </button>
+                    </div>
+                    <group col="4">
+                        <field name="name" string="Assortment Name"/>
+                        <field name="model_id" invisible="1"/>
+                        <field name="active"/>
+                        <field name="is_assortment" invisible="1"/>
+                    </group>
+                    <group>
+                        <field name="domain" widget="char_domain" options="{'model_field': 'model_id'}"/>
```

Comment:
@Laurent-Corron The widget to use is not properly set...

```suggestion
                        <field name="domain" widget="domain" options="{'model': 'product.product'}"/>
```

Reply:
fixed!